docs/components/cache.rst

```diff
@@ -101,17 +101,17 @@ Delivered adapters
 
 Redis configuration in ``local.php``:
 
-.. code-block:: php
-
-    'redis' => [
-        'dsn' => 'redis://localhost',
+.. code-block:: PHP
+    
+    'cache_adapter_redis' => [
```

Comment:
This is the correct key - thanks for the change! You're also correct about the memcached key as well, it should be `cache_adapter_memcached` - can you make that change and I'll approve?